contract_variable_qty_timesheet/README.rst

```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
+   :alt: License: AGPL-3
+
+===============================
+Contract Variable Qty Timesheet
+===============================
+
+This module extends the functionality of contract_variable_quantity adding
+several variable quantity formulas to allow to invoice lines from Timesheet
+(Analytic Lines)
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Go to Invoicing > Sales > Contracts and select or create a new contract.
+#. Check *Generate recurring invoices automatically*.
+#. Add a new recurring invoicing line.
+#. Select "Variable quantity" in column "Qty. type".
+#. Select "Project Timesheets", "Tasks Timesheets" or "Analytic Same Product"
+   depending on your need.
```

Comment:
- This 3 types of formulas are not obvious for a user, in my opinion I think we mus explain a little bit.
- There is a use case that make this not work: if you follow this steps you will never have a Project created automatically so this doesn't work.

Reply:
> There is a use case that make this not work: if you follow this steps you will never have a Project created automatically so this doesn't work.

This is out of scope of this module.